packages/beacon-node/src/chain/archiver/archiveBlocks.ts

```diff
@@ -73,28 +71,11 @@ export async function archiveBlocks(
     });
 
     if (finalizedPostDeneb) {
-      await db.blobsSidecar.batchDelete(nonCanonicalBlockRoots);
+      await db.blobSidecars.batchDelete(nonCanonicalBlockRoots);
       logger.verbose("Deleted non canonical blobsSider from hot DB");
     }
   }
 
-  // Delete expired blobs
-  // Keep only `[max(GENESIS_EPOCH, current_epoch - MIN_EPOCHS_FOR_BLOBS_SIDECARS_REQUESTS), current_epoch]`
-  if (finalizedPostDeneb) {
-    const blobsSidecarMinEpoch = currentEpoch - config.MIN_EPOCHS_FOR_BLOBS_SIDECARS_REQUESTS;
```

Comment:
Where are blobs pruned now?

Reply:
there is no `blobsSidecarArchive` as they are just deleted using `pruneBlobSidecarsFromHotDb` (instead of earlier doing `migrateBlobsSidecarFromHotToColdDb`) over here: https://github.com/ChainSafe/lodestar/pull/5181/files#diff-31bca7e499ec6e9d38d9ce318c6d239d35fa4f6b687213f84b75da217ac2415bR131

Reply:
But without blobsSidecarArchive, do we persist blobs from finalized blocks?

Reply:
no, we don't we just prune them out of hot db

Reply:
How are you gonna serve this ReqResp requests? https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/p2p-interface.md#blobsidecarsbyrange-v1 you need to serve finalized blobs

Reply:
right, i mistook that even for by range requests, finalized blobs need not be served
I will re-add the functionality